Regression/WarpX-tests.ini

```diff
@@ -2606,7 +2606,23 @@ analysisRoutine = Examples/Modules/ParticleBoundaryProcess/analysis_reflection.p
 buildDir = .
 inputFile = Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py
 runtime_params =
-customRunCmd = python PICMI_inputs_2d.py
+customRunCmd = python PICMI_inputs_2d.py 0
```

Comment:
Let's add a named parameter here, so it becomes self-describing here. Please see above :)